tfm/repro/TestRepro.cxx

```diff
@@ -322,6 +262,8 @@ TestRepro::TestRepro(const resip::Data& name,
 
 TestRepro::~TestRepro()
 {
+   mQValueTargetHandler = nullptr;
```

Comment:
We should be setting mQValueTargetHandler to nullptr in the constructor as well. :)

Reply:
Also, it's not really necessary to do it in the destructor, but I'm fine with leaving it there.